client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedSettings from 'my-sites/exporter/advanced-settings';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	propTypes: {
+		advancedSettings: PropTypes.shape( {
+			isVisible: PropTypes.bool.isRequired
+		} )
```

Comment:
Good point, I thought `isVisible.isRequired` would cover it but it looks like it won't if `advancedSettings` isn't included. I've added these in the latest push.